### PR TITLE
fix issue user request check in not admin

### PR DESCRIPTION
**Pull request by nnhathung**
Fixes #

#### Short description of what this resolves:
- error when user check in not super admin

#### Changes proposed in this pull request:

- remove validate when user check in

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Review by cweitat — DISMISSED** [review]

File: app/api/user_check_in.py [hunk 1]
```diff
@@ -77,31 +76,16 @@ def before_post(_args, _kwargs, data):
         :return:
         """
         require_relationship(['station'], data)
-        if not has_access('is_coorganizer', station=data.get('station')):
```
Comment (cweitat) [inline app/api/user_check_in.py:80]: so removing this means guest can also check in themselves? supposed to be only admin/organiser/coorganiser of event only able to chck in

Comment (cweitat) [reply app/api/user_check_in.py:80]: @nnhathung

Comment (nnhathung) [reply app/api/user_check_in.py:80]: @cweitat updated, only admin/organiser/coorganiser of event can do checkin

**Review by cweitat — APPROVED** [review]
LGTM; approving to merge for test

**Comment by mariobehling** [comment]
Thanks. Please resolve Deepsource Python issues. More details here https://circleci.com/gh/fossasia/open-event-server/25472

**Review by mariobehling — APPROVED** [review]

**Comment by mariobehling** [comment]
DeepSource still failing.

**Review by cweitat — APPROVED** [review]

**Pull request merged**